src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/_UnitTest/test_spinningBodyOneDOFStateEffector.py

```diff
@@ -42,9 +42,7 @@
 # uncomment this line if this test has an expected failure, adjust message as needed
 # @pytest.mark.xfail() # need to update how the RW states are defined
 # provide a unique test method name, starting with test_
-
-
-def test_spinningBodyTest(show_plots):
+def spinningBodyTest(show_plots, cmdTorque, lock):
```

Comment:
undo this, this is the method that should start with `test_`.  It is the only method that has the assert statement.

Reply:
Done.